tests/common/logout_test.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var App = require('../lib/app');
+var assert = require('assert');
+
+describe('Logout', function() {
+  var app;
+  var loginView;
+  var servicesView;
+  this.timeout(30000);
+
+  before(() => {
+    app = new App();
+    return app.init()
+      .then(defaultView => { loginView = defaultView; });
+  });
+
+  after(() => {
+    return app.stop();
+  });
+
+  it('should logout', () => {
+    return loginView.loginSuccess(12345678).then(function(servicesView) {
```

Comment:
So it cannot be nested inside the arrow funtion?
Would this be valid, changing how to start the test?
it('should logout', function() {

Otherwise I do not know how not to nest the .then...:S